Providers/AdjustProvider.m

```diff
@@ -41,6 +41,14 @@ - (void)event:(NSString *)event withProperties:(NSDictionary *)properties {
     ADJEvent *const adjustEvent = [ADJEvent eventWithEventToken:event];
 
     [properties enumerateKeysAndObjectsUsingBlock:^(id key, id value, BOOL *stop) {
+        // Adjust assumes value is a string, so if it's not convert it to a string value
+        if (![value isKindOfClass:[NSString class]]) {
+            if ([value respondsToSelector:@selector(stringValue)]) {
+                value = [value stringValue];
+            } else {
+                value = [value description];
```

Comment:
Just for completeness: this is also only true for NSObjects/CustomStringConvertible in Swift. If you insert a Swift class that does not inherit from NSObject or implements CustomStringConvertible, this would still crash.
But it should be ok how it is ;)